docs/src/community/phrasebook.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: ../common_links.inc
+
+.. _phrasebook:
+
+Package Phrasebook
+===================
+
+There are a number of similar packages to Iris, and a lot of these have their own
+terminology for similar things. Whether you're coming or going, we hope this might
+be a helpful guide to these differences!
+Definitions for each can be found in :ref:`glossary`. See also
+`Xarray terminology <https://docs.xarray.dev/en/stable/user-guide/terminology.html>`_.
+
+.. list-table:: Phrasebook
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - Iris
+     - Xarray
+     - Example
+     - Notes
+   * - Non-Lazy
+     - Eager
+     -
+     - Used to relate to functions, rather than the data.
+   * - Cube
+     - DataArray
+     -
+     -
+   * - Cubelist
+     - Dataset
+     -
+     -
```

Comment:
It's worth noting that `Datasets` are quite a a bit richer in their data model than `CubeList`s, which are effectively just a collection of objects with some methods attached. I think the major difference from what I can gather is that coordinates are a proper member of a `Dataset` while for a `CubeList`, a coordinate can only be accessed via one of the `Cubes` contained within it. I think in this table it's worth just noting that a `CubeList` is a much simpler object and the correspondance is not exact.